extras/rya.geoindexing/src/test/java/org/apache/rya/indexing/mongo/MongoGeoIT.java

```diff
@@ -0,0 +1,153 @@
+        assertEquals(4, client.getDatabase("indexerTests").getCollection("rya_rya_geo").count());
+        assertEquals(1, client.getDatabase("indexerTests").getCollection("rya_rya_temporal").count());
+        assertEquals(2, client.getDatabase("indexerTests").getCollection("rya_rya_freetext").count());
+
+        //free text -- remove one from many
+        String delete = "DELETE DATA \n" //
+           + "{\n"
+           + "  <urn:people> <http://www.w3.org/2000/01/rdf-schema#label> \"Alice Palace Hose\" "
+           + "}";
+        Update update = conn.prepareUpdate(QueryLanguage.SPARQL, delete);
+        update.execute();
+
+        // temporal -- remove one from one
+        delete = "DELETE DATA \n" //
+           + "{\n"
+           + "  <foo:time> <Property:atTime> \"0001-02-03T04:05:06Z\" "
+           + "}";
+
+        update = conn.prepareUpdate(QueryLanguage.SPARQL, delete);
+        update.execute();
+
+        assertEquals(0, client.getDatabase("indexerTests").getCollection("rya_rya_temporal").count());
+        assertEquals(1, client.getDatabase("indexerTests").getCollection("rya_rya_freetext").count());
+        assertEquals(6, client.getDatabase("indexerTests").getCollection("rya__triples").count());
+    }
+
+    private void populateRya() throws Exception {
+        final ValueFactory VF = new ValueFactoryImpl();
+        // geo 2x2 points
+        final GeometryFactory GF = new GeometryFactory();
+        for (int x = 0; x <= 1; x++) {
+            for (int y = 0; y <= 1; y++) {
+                final Geometry geo = GF.createPoint(new Coordinate(x + .5, y + .5));
+                final RyaStatement stmnt = statement(geo);
+                final Statement statement = RyaToRdfConversions.convertStatement(stmnt);
+                conn.add(statement);
+            }
+        }
+
+        // freetext
+        final URI person = VF.createURI("http://example.org/ontology/Person");
+        String uuid;
+
+        uuid = "urn:people";
+        conn.add(VF.createURI(uuid), RDF.TYPE, person);
+        conn.add(VF.createURI(uuid), RDFS.LABEL, VF.createLiteral("Alice Palace Hose", VF.createURI("http://www.w3.org/2001/XMLSchema#string")));
+
+        uuid = "urn:people";
+        conn.add(VF.createURI(uuid), RDF.TYPE, person);
+        conn.add(VF.createURI(uuid), RDFS.LABEL, VF.createLiteral("Bob Snob Hose", "en"));
+
+        // temporal
+        final TemporalInstant instant = new TemporalInstantRfc3339(1, 2, 3, 4, 5, 6);
+        final URI time = VF.createURI("Property:atTime");
+        conn.add(VF.createURI("foo:time"), VF.createURI("Property:atTime"), VF.createLiteral(instant.toString()));
+    }
+
+    private static RyaStatement statement(final Geometry geo) {
+        final ValueFactory vf = new ValueFactoryImpl();
+        final Resource subject = vf.createURI("urn:geo");
+        final URI predicate = GeoConstants.GEO_AS_WKT;
+        final Value object = vf.createLiteral(geo.toString(), GeoConstants.XMLSCHEMA_OGC_WKT);
```

Comment:
okay, I re-read your earlier comment.  no geo test is fine for this PR since it's coming in another PR.